### ISSUE TITLE
nanodu is failing to submit PR against 3rd party repos

### ISSUE THREAD

**Issue by josesimoes** [nanoframework/Home#1142]
### Tool

nanodu

### Description

The tool is failing to submit the PR with the updates against a 3rd party repo.

### How to reproduce

See log from this github action run: https://github.com/nanoframework/amqpnetlite/actions/runs/3054202425

**Comment by josesimoes** [comment]
@torbacz any ideas on what could be happening?

**Comment by josesimoes** [comment]
That's clearly something with the fact that's a fork from "upstream". I did add the repoOwner parameter and that's being passed correctly, so it has to be something with the API call that creating the PR...

**Comment by torbacz** [comment]
@josesimoes 
Just checked log again. 
![image](https://user-images.githubusercontent.com/19893965/190260982-4557e226-4e65-4683-8d20-bfcbfd99f2b9.png)

Shouldn't it be "nanoframework"?

**Comment by josesimoes** [comment]
Nope. Because the PR is meant to be submitted against the upstream repo.
